src/test/java/org/apache/commons/collections4/map/AbstractMapTest.java

```diff
@@ -134,6 +141,13 @@
         JDK12 = str.startsWith("1.2");
     }
 
+    /**
+     * Flag to indicate the collection makes no ordering guarantees for the iterator. If this is not used
+     * then the behaviour is assumed to be ordered and the output order of the iterator is matched by
+     * the toArray method.
+     */
+    protected static final int UNORDERED = 0x1;
```

Comment:
Remove this. The property from AbstractCollectionTest should be used as it is configuring the test within that class.